manifests/params.pp

```diff
@@ -29,6 +29,9 @@
 
       $path_separator = ':'
     }
+    'windows' : {
+
```

Comment:
Everything is handled in the provider.  We could do the source here but doesn't make sense when we have a type and provider to handle the logic